flow-jcloud.yml

```diff
@@ -30,10 +30,9 @@ executors:
     uses_with:
       name: ViT-L-14-336::openai
     replicas: 1
-    host: 'demo-cas.jina.ai'
+    host: 'api.clip.jina.ai'
     port: 2096
-    tls: true
-    external: true
+    external: false
```

Comment:
why is not true?

Reply:
And, in the current implementation, the clip-as-service is used defaultly as external executor. The user is not aware of this behavior. I think it would be better to inform the user how to use clip-as-service, and what's clip-as-service can offer.

Reply:
I am not sure to understand why external is false when using CAS as an external Executor

Reply:
> why external is false when using CAS as an external Executor

@samsja external is false since we won't use CAS external executor default. To use it, we must provide the acess token in the flow.

Reply:
Back then, if executor config has `host & port` configured, Flow was still using that host even when `external` is `false`. I am not if it is still works as now.

https://jinaai.slack.com/archives/C018F60RBL5/p1660060726395409
https://github.com/jina-ai/jina/issues/5052#issuecomment-1210478687

Reply:
If this is true, I would say `external` is not a good design. However, I will remove the `host & port`. Thanks for your information, BTW.

Reply:
@delgermurun I have removed `host & port` in the default yaml file. Please take a review again.

Reply:
how about comment out (with some note/instructions) instead of deleting?